atom/browser/api/atom_api_web_contents.cc

```diff
@@ -527,12 +527,14 @@ bool WebContents::DidAddMessageToConsole(content::WebContents* source,
 
 void WebContents::OnCreateWindow(
     const GURL& target_url,
+    const content::Referrer& referrer,
     const std::string& frame_name,
     WindowOpenDisposition disposition,
     const std::vector<std::string>& features,
     const scoped_refptr<content::ResourceRequestBody>& body) {
   if (type_ == BROWSER_WINDOW || type_ == OFF_SCREEN)
-    Emit("-new-window", target_url, frame_name, disposition, features, body);
+    Emit("-new-window", target_url, referrer, frame_name, disposition, features,
```

Comment:
Although it's not technically public API, we can try to not break people using it _for whatever reason_.

Just making this new `referrer` arg the last arg in the list should make this 🆗